src/main/java/com/yahoo/sketches/theta/DirectIntersection.java

```diff
@@ -88,44 +88,44 @@
       throw new IllegalArgumentException("PreambleLongs must = 3.");
     }
     int serVer = extractSerVer(pre0);
-    if (serVer != 3) throw new IllegalArgumentException("Ser Version must = 3");
+    if (serVer != 3) {
+      throw new IllegalArgumentException("Ser Version must = 3");
+    }
     int famID = extractFamilyID(pre0);
     Family.INTERSECTION.checkFamilyID(famID);
     //Note: Intersection does not use lgNomLongs or k, per se.
-    lgArrLongs_ = extractLgArrLongs(pre0); //current hash table size
-    maxLgArrLongs_ = checkMaxLgArrLongs(srcMem);
+    lgArrLongs = extractLgArrLongs(pre0); //current hash table size
+    maxLgArrLongs = checkMaxLgArrLongs(srcMem);
     int flags = extractFlags(pre0);
-    empty_ = (flags & EMPTY_FLAG_MASK) > 0;
+    empty = (flags & EMPTY_FLAG_MASK) > 0;
     
-    seedHash_ = computeSeedHash(seed);
+    seedHash = computeSeedHash(seed);
     short seedHashMem = srcMem.getShort(SEED_HASH_SHORT);
-    Util.checkSeedHashes(seedHashMem, seedHash_); //check for seed hash conflict
+    Util.checkSeedHashes(seedHashMem, seedHash); //check for seed hash conflict
     
-    curCount_ = srcMem.getInt(RETAINED_ENTRIES_INT);
-    thetaLong_ = srcMem.getLong(THETA_LONG);
+    curCount = srcMem.getInt(RETAINED_ENTRIES_INT);
+    thetaLong = srcMem.getLong(THETA_LONG);
     
-    if (empty_) {
-      if (curCount_ != 0) {
-        throw new IllegalArgumentException(
-            "srcMem empty state inconsistent with curCount: "+empty_+","+curCount_);
-      }
-      //empty = true AND curCount_ = 0: OK
+    if (empty && curCount != 0) {
```

Comment:
This is the only change in this entire file that potentially has merit.  However, what was missed is the value of the comments embedded in the compound if/then/else clause:  The original code plus comments was specifically structured to help the reader understand subtleties in the code flow.  The comments were strategically placed in the position of the else clauses even though the else clauses are absent.  The suggested change ignores the value of the comments and in the suggested form the internal comment is now incorrect and would be confusing to the reader!  The compiler and JIT optimizer would restructure this compound statement anyway so there is NO performance impact on this change.  This change is rejected.